### PR TITLE
Feat/access checking

### PR DESCRIPTION
**Pull request by prodislav**
https://trello.com/c/zLeo4RfF/233-initiailize-admin-console

### CODE REVIEW

File: frontend/src/store/auth/auth.action.ts [hunk 1]
```diff
@@ -16,6 +16,8 @@ export enum AuthTypes {
   RegistrationError = '[auth] Registration (Error)',
   SocialNetworksLogin = '[auth] Social Networks Login',
   LoginError = '[auth] Login (error)',
+  GetUserLinks = '[auth] Get User Links (error)',
```
Comment (DrJekill352) [inline frontend/src/store/auth/auth.action.ts]: Wrong message

File: frontend/src/store/auth/auth.effects.ts [hunk 2]
```diff
@@ -90,19 +92,46 @@ export const setCurrentUser$ = (action$: ActionsObservable<SetCurrentUser>) =>
       const user: FrontEndUser | undefined = action.payload;
       if (user) {
         store.dispatch(new SetLanguage(user.email));
+        console.log('AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA', user.id);
```
Comment (DrJekill352) [inline frontend/src/store/auth/auth.effects.ts]: remove

File: server/models/role.ts [hunk 3]
```diff
@@ -28,3 +31,8 @@ export enum Roles {
     Admin = 'admin',
     User = 'user'
 }
+
+export enum RolesId {
+    User = 1,
```
Comment (DrJekill352) [inline server/models/role.ts]: remove please

File: frontend/src/store/auth/auth.effects.ts [hunk 4]
```diff
@@ -8,16 +8,17 @@ import { ErrorBlock, SnackbarType, UserFieldsToLogin } from 'models';
...
 import {
   AuthTypes,
+  GetUserLinks,
+  GetUserLinksSuccess,
   LoginError,
   LoginUser,
   LogoutUser,
   RegisterUser,
   RegistrationError,
-  RegistrationSuccess,
```
Comment (DrJekill352) [inline frontend/src/store/auth/auth.effects.ts]: Why are you remove success action?

**Comment by DrJekill352** [comment]
And check console.log, again. Please

File: admin/src/App.test.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { App } from './App';
+
+it('renders without crashing', () => {
```
Comment (DrJekill352) [inline admin/src/App.test.tsx]: You forget 'describe' before 'it'

Comment (prodislav) [reply admin/src/App.test.tsx]: added

File: frontend/src/store/auth/auth.effects.ts (hunk 4 above)
Comment (prodislav) [reply frontend/src/store/auth/auth.effects.ts]: added

File: server/models/role.ts (hunk 3 above)
Comment (prodislav) [reply server/models/role.ts]: still use

File: frontend/src/store/auth/auth.action.ts (hunk 1 above)
Comment (prodislav) [reply frontend/src/store/auth/auth.action.ts]: changed

**Pull request merged**